modules/text2vec-openai/ent/vectorization_result.go

```diff
@@ -45,6 +47,13 @@ func GetRateLimitsFromHeader(header http.Header, isAzure bool) *modulecomponents
 	updateWithMissingValues := false
 	if limitRequests < 0 || limitTokens < 0 || remainingRequests < 0 || remainingTokens < 0 {
 		updateWithMissingValues = true
+
+		// logging all headers as there should not be anything sensitive according to the documentation:
+		// https://platform.openai.com/docs/api-reference/debugging-requests
+		l.WithSampling(func(l logrus.FieldLogger) {
+			l.WithField("headers", header).
+				Warn("rate limit headers are missing or invalid, using dummy")
```

Comment:
we are not using a dummy value, but will skip updating the the value in the batching algo